GetCertificate/printer.ts

```diff
@@ -182,8 +182,9 @@ export const printUvci = (
  * @param lang the preferred language
  * @returns a string containing the Info markdown
  */
-export const printExpiredInfo = (lang: o.Option<PreferredLanguage>): string =>
-  getPrinterForLanguage(lang).expiredInfoPrinter();
+export const printExpiredOrRevokedInfo = (
```

Comment:
I'd rather not have a printer function that hides some kind of decisions. In my opinion is better to have separated printer function for each case, and let the aggregator function (the handler in our case) to explicitly choose what to use.

Specifically, I'd probably go with
```ts
export const printExpiredInfo = (lang: o.Option<PreferredLanguage>): string =>
  getPrinterForLanguage(lang).expiredInfoPrinter();

export const printRevokedInfo = (lang: o.Option<PreferredLanguage>): string =>
  getPrinterForLanguage(lang).revokedInfoPrinter();
```
and let the language file handle the specific string. Are they the same value? No problem, it's incidental 🤷

Reply:
We are actually saying it [here](https://github.com/pagopa/io-functions-eucovidcerts/blob/4cb8b7305794a0dde3e356414d97ba3f610dc2de/GetCertificate/markdown/eucovidcertExpiredInfoEn.ts#L6): _"Your certificate in invalid: it may have expired or been temporarily revoked"_ 
That's why I though to unify the functions.

Reply:
You are right, the main problem is we cannot determine the state of the Certificate. Maybe we could have occur with a `UnknownCertificate` type, in which to include every unknown responses from DGC. I guess is too late now, maybe it's a lesson for next projects.